docs/customize/Import-Styles.md

```diff
@@ -1,4 +1,4 @@
-# Configuring the Import of OSM data
+\# Configuring the Import of OSM data
```

Comment:
What does that do?

Reply:
It shows how easy it is to hit the \ key by accident. (removed now)